tuxemon/states/combat/combat.py

```diff
@@ -323,6 +323,7 @@ def draw(self, surface: pygame.surface.Surface) -> None:
     def draw_hp_bars(self) -> None:
         """Go through the HP bars and redraw them."""
         for monster, hud in self.hud.items():
+            assert hud.image
```

Comment:
This method is going to be called at every game iteration, do we really assume `hud.image` can be changed to null at any point?

If there is no external element which can modify the `hud.image`, then I would suggest to do this check at instance creation instead.